src/app/clients/getClientsTableData.ts

```diff
@@ -57,25 +59,26 @@ const getProcessingData = async () => {
 export const processingDataToClientsTableData = async (
     processingData: ProcessingData
 ): Promise<ClientsTableRow[]> => {
-    const clientTableRows = [];
+    const clientTableRows: ClientsTableRow[] = [];
     const congestionChargeDetails = await getCongestionChargeDetails(processingData);
 
     for (let index = 0; index < processingData.length; index++) {
         const parcel = processingData[index];
         const client = parcel.client!;
 
         clientTableRows.push({
+            primaryKey: client.primary_key,
             parcelId: parcel.parcel_id,
             flaggedForAttention: client.flagged_for_attention,
             requiresFollowUpPhoneCall: client.signposting_call_required,
             fullName: client.full_name,
             familyCategory: familyCountToFamilyCategory(client.family.length),
             addressPostcode: client.address_postcode,
             collectionCentre: parcel.collection_centre ?? "-",
-            collectionDatetime: parcel.collection_datetime,
             congestionChargeApplies: congestionChargeDetails[index].congestionCharge,
             packingDate: formatDatetimeAsDate(parcel.packing_datetime),
             packingTimeLabel: datetimeToPackingTimeLabel(parcel.packing_datetime),
+            collectionDatetime: parcel.collection_datetime ?? "-",
```

Comment:
why the reorganisation?

Reply:
there was a bug in Actions and status bar which was on main - the collectionDatetime prop was called there but was not defined here, so i just added it (for lint_fix to work)